docs/source/about/changelog.rst

```diff
@@ -23,7 +23,13 @@ more info, see the :ref:`Contributor Guide <Creating a Changelog Entry>`.
 Unreleased
 ----------
 
-No changes.
+**Reverted**
```

Comment:
Just realized we're using a non-standard `Keep-A-Changelog` category here. Not a big deal though.